internal/types/resource/resource.go

```diff
@@ -4,22 +4,24 @@ package resource
 type Type uint
 
 const (
-	Unknown     Type = 0
-	All         Type = 1
-	Scope       Type = 2
-	User        Type = 3
-	Group       Type = 4
-	Role        Type = 5
-	AuthMethod  Type = 6
-	Account     Type = 7
-	AuthToken   Type = 8
-	HostCatalog Type = 9
-	HostSet     Type = 10
-	Host        Type = 11
-	Target      Type = 12
-	Controller  Type = 13
-	Worker      Type = 14
-	Session     Type = 15
+	Unknown           Type = 0
+	All               Type = 1
+	Scope             Type = 2
+	User              Type = 3
+	Group             Type = 4
+	Role              Type = 5
+	AuthMethod        Type = 6
+	Account           Type = 7
+	AuthToken         Type = 8
+	HostCatalog       Type = 9
+	HostSet           Type = 10
+	Host              Type = 11
+	Target            Type = 12
+	Controller        Type = 13
+	Worker            Type = 14
+	Session           Type = 15
+	CredentialStore   Type = 16
+	CredentialLibrary Type = 16
```

Comment:
```suggestion
	CredentialLibrary Type = 17
```

Reply:
Hmmm, I thought it was because they were somehow stored in the DB but after looking closer I don't think that is true.  I thought I had asked about it before but maybe I'm thinking of a different enum.  I've updated these to use iota. @jefferai let me know if there is a reason to not use iota here.